docker/demo/presto-batch1.commands

```diff
@@ -0,0 +1,6 @@
+select symbol, max(ts) from stock_ticks_cow group by symbol HAVING symbol = 'GOOG';
+select symbol, max(ts) from stock_ticks_mor group by symbol HAVING symbol = 'GOOG';
+select symbol, max(ts) from stock_ticks_mor_rt group by symbol HAVING symbol = 'GOOG';
```

Comment:
@nsivabalan presto does not support mor_rt tables yet. Meaning it can produce unreliable result since currently, it can only read the base parquet file (just supports Read optimized view). Can we remove the presto queries against *mor_rt table to avoid confusion?

Reply:
I do think this needs to be added back when presto supports rt views.

Reply:
+1